test/dotnet-run.Tests/RunTests.cs

```diff
@@ -20,7 +20,7 @@ public void RunsSingleTarget()
             TestInstance instance = TestAssetsManager.CreateTestInstance(Path.Combine(RunTestsBase, "TestAppFullClr"))
```

Comment:
How do I cover it? Muxer filters out this argument so i can't just write line, and I don't want to mock RunCommand, this it blocking issue for ASP.NET.